Frameworks/Security/GenericPasswordItemHandler.mm

```diff
@@ -589,16 +558,14 @@ - (OSStatus)remove:(NSDictionary*)queryDictionary {
         return errSecParam;
     }
 
-    // TODO: Error handling when projections support it.
-    NSArray* credArray = [_vault retrieveAll];
+    WFC::IVectorView<PasswordCredential> credArray = _vault.RetrieveAll();
 
     // Documentation for remove states that it by default removes all matching rows while documentation
     // for query defaults to a single item. Go figure.
-    std::vector<StrongId<WSCPasswordCredentialWrapper>> credentialsToRemove = FindMatchingCredentials(queryDictionary, credArray, true);
+    std::vector<StrongId<PasswordCredentialWrapper>> credentialsToRemove = FindMatchingCredentials(queryDictionary, credArray, true);
 
     for (const auto& credential : credentialsToRemove) {
-        // TODO: Error handling when projections support it.
```

Comment:
>// TODO: Error handling when projections support it. [](start = 8, length = 52)

Is crashing on failure considered 'error handling'?  do we need a separate issue tracking properly handling errors?